### PR TITLE
Complete task 3

### PR DESCRIPTION
**Pull request by Oigen43**
All tasks was done 11/11

Link to application: https://d5szlxe6wew8g.cloudfront.net/
Link to product-service API:  https://bmac37yjm8.execute-api.us-east-1.amazonaws.com/dev/products
Link to FE pull request: https://github.com/Oigen43/cloud-shop-web/pull/2
Swagger: https://app.swaggerhub.com/apis-docs/Oigen43/cloud-shop/0.1

### CODE REVIEW

File: product-service/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "babel-polyfill": "^6.23.0",
+    "babel-preset-env": "^1.6.0",
+    "serverless-webpack": "^5.3.1",
+    "webpack": "^4.35.2"
+  },
+  "author": "The serverless webpack authors (https://github.com/elastic-coders/serverless-webpack)",
+  "license": "MIT",
+  "dependencies": {
+    "axios": "^0.21.0",
+    "jest": "^26.6.1"
```
Comment (ezhukauskas) [inline product-service/package.json:22]: I think this is more like a dev dependency

File: product-service/src/handlers/get-product-by-id.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+export const getProductById = async (event) => {
+  try {
+    const { productId } = event.pathParameters;
+    const product = productList.find(item => item.id === productId);
+
+    // just a random request for making the task with async/await
+    await axios.get('https://jsonplaceholder.typicode.com/todos/1');
+
+    if (!product) throw new Error();
```
Comment (ezhukauskas) [inline product-service/src/handlers/get-product-by-id.js:13]: Why? Why not just return a 404 response? Furthermore, you will always get 404 if an error is thrown at another level, e.g. in DB.

**Review by ezhukauskas — APPROVED** [review]
Good job!

File: product-service/package.json (hunk 1 above)
Comment (Oigen43) [reply product-service/package.json:22]: you are right, thx, i will change it after crosscheck

File: product-service/src/handlers/get-product-by-id.js (hunk 2 above)
Comment (Oigen43) [reply product-service/src/handlers/get-product-by-id.js:13]: yes, it could be done better :)

**Pull request merged**